### PR TITLE
ME14: Synced SVD Files to register files. Fixed SVDConv errors.

### PR DESCRIPTION
**Pull request by sihyung-maxim**
Changes:
- Updated ME14 register files to pass linter.
- Updated copyright notices.
- Updated max32665.svd.

### CODE REVIEW

File: Libraries/PeriphDrivers/Source/WUT/wut_reva_me14.svd [hunk 1]
```diff
@@ -45,7 +45,21 @@
         </fields>
       </register>
       <register>
-        <name>INTR</name>
+        <name>PWM</name>
```
Comment (lorne-maxim) [inline Libraries/PeriphDrivers/Source/WUT/wut_reva_me14.svd:48]: The WUT only supports one-shot, continuous, and compare modes.  Many of these additional fields are not available.

**Review by lorne-maxim — CHANGES_REQUESTED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32665/Include/rtc_regs.h [hunk 2]
```diff
@@ -113,8 +129,8 @@ typedef struct {
  *           when this register rolls over from 0xFF to 0x00.
  * @{
  */
- #define MXC_F_RTC_SSEC_SSEC_POS                        0 /**< SSEC_SSEC Position */
- #define MXC_F_RTC_SSEC_SSEC                            ((uint32_t)(0xFFFUL << MXC_F_RTC_SSEC_SSEC_POS)) /**< SSEC_SSEC Mask */
+#define MXC_F_RTC_SSEC_SSEC_POS                        0 /**< SSEC_SSEC Position */
+#define MXC_F_RTC_SSEC_SSEC                            ((uint32_t)(0x7FFUL << MXC_F_RTC_SSEC_SSEC_POS)) /**< SSEC_SSEC Mask */
```
[the remark this answers is not in the document]
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32665/Include/rtc_regs.h]: Yes, you're correct, nice catch.

File: Libraries/PeriphDrivers/Source/LP/pwrseq_me14.svd [hunk 3]
```diff
@@ -44,7 +44,13 @@
             </enumeratedValues>
           </field>
           <field>
-            <name>BKGRND</name>
+            <name>BLKDET</name>
```
[the remark this answers is not in the document]
Comment (sihyung-maxim) [reply Libraries/PeriphDrivers/Source/LP/pwrseq_me14.svd]: BLKDET was deleted, that field should not have been added. However, the SDMAWKST and SDMAWKEN fields were missing from our register file but were added by design.

File: Libraries/PeriphDrivers/Source/WUT/wut_reva_me14.svd (hunk 1 above)
Comment (sihyung-maxim) [reply Libraries/PeriphDrivers/Source/WUT/wut_reva_me14.svd]: Thanks. I believe all the parts supporting WUT still contain the PWM-related fields in their register files. I'll go ahead and update those files.

**Review by lorne-maxim — APPROVED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32665/Include/aes_key_regs.h [hunk 4]
```diff
@@ -1,115 +0,0 @@
...
- * The mere transfer of this software does not imply any licenses
- * of trade secrets, proprietary technology, copyrights, patents,
- * trademarks, maskwork rights, or any other form of intellectual
- * property whatsoever. Maxim Integrated Products, Inc. retains all
- * ownership rights.
- *
- *
- *************************************************************************** */
-
-#ifndef _AES_KEY_REGS_H_
```
Comment (Jacob-Scheiffler) [inline Libraries/CMSIS/Device/Maxim/MAX32665/Include/aes_key_regs.h:40]: Should these regs have been deleted? I see they are still in UG and are referenced as cipher key sources in TPU section as well.

**Review by Jacob-Scheiffler — APPROVED** [review]

**Review by lorne-maxim — APPROVED** [review]

**Pull request merged**